packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -144,7 +146,7 @@ class _DaemonServer {
   }
 }
 
-typedef CommandHandler = Future<dynamic> Function(Map<String, dynamic> args);
+typedef CommandHandler = Future<dynamic> Function(Map<String, dynamic> args, Stream<List<int>> binary);
```

Comment:
This is a great idea! Just added a new callback table. Also added assertion so that the same method will not be registered on both callback tables. Thanks!